handbook/security-policies.md

```diff
@@ -215,6 +215,136 @@ Fleet policy requires that:
+| Employees, contractors, temporary workers, etc. | Acting at all times in a manner which does not place at risk the security of themselves, colleagues, and of the information and resources they have use of<br/>  Helping to identify areas where risk management practices should be adopted<br/>  Adhering to company policies and standards of conduct Reporting incidents and observed anomalies or weaknesses                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| Head of people operations                       | Ensuring employees and contractors are qualified and competent for their roles<br/>  Ensuring appropriate testing and background checks are completed<br/>  Ensuring that employees and relevant contractors are presented with company policies <br/>  Ensuring that employee performance and adherence to values is evaluated<br/>  Ensuring that employees receive appropriate security training                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| Head of business operations                     | Responsible for oversight over third-party risk management process Responsible for review of vendor service contracts                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
fyi @GuillaumeRoss no pressure either way, but in case it helps you:  you can safely omit the trailing `|` on each line in GitHub-flavored markdown tables.  Also the column headers don't have to actually line up with each column

cc @Desmi-Dizney

Reply:
@mikermcneil I cheat by using TableFlip, the greatest macOS application ever made - I would never take the time to make them like this manually!